src/Gui/Application.cpp

```diff
@@ -344,7 +344,7 @@ namespace {
         std::stringstream str;
         str << "Image formats (";
         for (const auto& ext : supportedFormats) {
-            str << "*." << ext.constData() << " ";
+        str << "*." << ext.constData() << " *." << ext.toUpper().constData() << " ";
```

Comment:
the indentation should have been kept I think

Reply:
Yes, I didn't notice.